playground/envs/.env

```
@@ -0,0 +1,3 @@
+VITE_APP_TITLE="UNH-Playground"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)